spec/que/job.bulk_enqueue_spec.rb

```diff
@@ -278,21 +296,13 @@ def assert_enqueue(
     end
   end
 
-  it "should raise when job_options are passed to .enqueue rather than .bulk_enqueue" do
-    assert_raises_with_message(Que::Error, "When using .bulk_enqueue, job_options must be passed to that method rather than .enqueue") do
```

Comment:
We should also update the [bulk enqueue documentation](https://github.com/que-rb/que/tree/master/docs#enqueueing-jobs-in-bulk) to remove the applicable limitations

Reply:
Will do.